core/controlplane/config/templates/cloud-config-worker

```diff
@@ -236,27 +257,38 @@ coreos:
             Environment="FLANNEL_IMAGE={{.FlannelImage.RktRepo}}"
             Environment="RKT_RUN_ARGS={{.FlannelImage.Options}} --uuid-file-save=/var/lib/coreos/flannel-wrapper2.uuid"
 {{end}}
+    {{- end}}
     - name: kubelet.service
       command: start
       runtime: true
       content: |
         [Unit]
+        {{ if not .Experimental.NetworkingDaemonSets.Enabled -}}
```

Comment:
- How about renaming `NetworkingDaemonSets` to e.g. [`SelfHostedCanal`](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md)? After reading the canal doc, I now believe we can safely call it so
- Can we move the new configuration(NetworkingDaemonSets before the my suggested rename) out of `.Experimental`? Sorry for the confusion but we're basically avoiding to use the `.Experimental` key anymore. We're in pre-1.0 and there is no strict rule about what's considered experimental or when it graduates from experimental and so on.
  - However, a note about the experimental feature in cluster.yaml comments would be welcomed!

Reply:
I didn't want to name it Canal as you can have selfhosted flannel if you don't supply UseCalico: true.  Perhaps another name better captures the hosted-in-kubernetes, removed-from-coreos-supplied-flannel-ness of the change?  Shall we call it KubeHostedNetworking, or SelfHostedNetworkFabric maybe?

I'll remove it from Experimental and make it top level?

Reply:
Thanks for pointing it out! Made sense a lot.

How about `Kubernetes.Networking.SelfHosting` then?

Probably we can also consider migrating other network-related keys like `useCalico` under that for more cohesion, in the future.

Reply:
Ok, I'll make those changes and re-push

Reply:
Err, did you want me to create a Kubernetes stanza?  If so should I move kubeDns, kubeProxy, kubernetesDashboard, + lots of others into it? (Feels quite a big change).

Or create it just for this settings?
Or could I get away with KubernetesNetworkingSelfHosting?